asmcli/lib/config.sh

```diff
@@ -26,7 +25,9 @@ configure_package() {
     kpt cfg set asm gcloud.compute.network "${GCE_NETWORK_NAME}"
   else
     kpt cfg set asm gcloud.core.project "${FLEET_ID}"
-    kpt cfg set asm anthos.servicemesh.controlplane.monitoring.enabled "false"
+    if ! is_cluster_registered; then
```

Comment:
Maybe checking CA is better? cluster registration will be required thus this condition will not be met. However, in the case of Citadel, within ASM source code, stackdriver will not be setup. Thus kpt cfg set asm anthos.servicemesh.controlplane.monitoring.enabled "true" in this case maybe misleading.

Reply:
Changed, thanks again.

Reply:
No problem!